_rules/no-auto-play-audio-80f0bf.md

```diff
@@ -0,0 +1,145 @@
+---
+id: 80f0bf
+name: Video or audio has no auto-play audio.
+rule_type: composite
+description: |
+    This rule checks that auto-play audio does not last for more than 3 seconds, or the audio has a control mechanism to stop or mute the it.
+accessibility_requirements:
+  wcag20:1.4.2: # Audio Control (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_rules:
+  - 95e043
+  - aaa1bf
+authors:
+  - Anne Thyme Nørregaard
+  - Bryn ANderson
+---
+
+## Applicability
+
+This rule applies to any HTML `<audio>`, `<video>` or `<source>` elements, with a `src` attribute referencing content with a duration of more than 3 seconds, that has an `autoplay` attribute equal to `true`, and that has both `paused` and `muted` attributes equal to `false`.
+
+## Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+- [Auto-play audio does not last more than 3 seconds](https://act-rules.github.io/rules/aaa1bf)
+- [Audio-only as a media alternative for text](https://act-rules.github.io/rules/95e043)
+ 
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+The native `<video>` and `<audio>` controls in several browser and assistive technology combinations are not keyboard accessible and the `<video>` or `<audio>` element itself may not be anounced. Authors are recomended to use custom controls for keyboard navigation and cross browser accessibility support in general.
+
+## Background
+
+- [Success Criterion 1.4.2 Audio Control](https://www.w3.org/TR/WCAG21/#audio-control)
```

Comment:
I think you should link to the understanding document (https://www.w3.org/WAI/WCAG21/Understanding/audio-control.html) not the SC, since that will be linked from the frontmatter (if I'm not mistaken)

Reply:
Changed the link to the "Understanding" one